Add Polygon ID page (#200)
verida/documentation: docs/extensions/polygon-id.md

[Check our User Guide](https://news.verida.io/user-guide-get-started-with-polygon-id-zero-knowledge-credentials-in-the-verida-wallet-f1d6f953f285) for more information and a walkthrough on how to use the Verida Wallet with Polygon ID.

## Credential Issuers and Verifiers

You can suggest your users to install and use the Verida Wallet to support your credential flows. The Verida Wallet is a mobile app available on iOS and Android and is free to use.

[<AppleAppStore className="appstorebutton" />](https://apps.apple.com/us/app/verida-vault/id1546599632)
[<GooglePlay className="appstorebutton" />](https://play.google.com/store/apps/details?id=io.verida.vault)

Feel free to [contact us](https://www.verida.network/ecosystem#partner) for any questions or partnership opportunities.

### Issuing a Polygon ID credential

Your application can issue a credential to a user in many different ways. A comprehensive list is available in the [Polygon ID Issuer documentation](https://0xpolygonid.github.io/tutorials/issuer/issuer-overview/) and includes; running an issuer node, using a javascript SDK or issuing on-chain via a smart contract.

> There is also a third party product, [Gatekeeper software](https://gatekeeper.software/), that can operate a Polygon ID node on your behalf and simplify the process of issuing credentials.

Author: ```suggestion
There are also third party products, such as [Gatekeeper software](https://gatekeeper.software/), that can operate a Polygon ID node on your behalf and simplify the process of issuing credentials.
```

Regardless of the process used to issue a credential, the end result will be the generation of a QR code that can be signed by the user to “receive” their credential.

> This QR code can be scanned by the Verida Wallet which will automatically detect it as a Polygon ID credential and prompt the user to save to their Wallet. This credential is encrypted and backed up on the self-sovereign Verida network, ensuring the user can easily restore their credentials on different devices, provided they backup their Verida Wallet seed phrase.

Author: ```suggestion
The credential offer can be transferred via QR code or deep link that the Verida Wallet will automatically detect as a Polygon ID credential and prompt the user to save to their Wallet. This credential is encrypted and backed up on the self-sovereign Verida Network, ensuring the user can easily restore their credentials on different devices, provided they backup their Verida Wallet seed phrase.
```

### Verifying a Polygon ID credential

Your application can verify a zero knowledge proof generated from a Polygon ID credential stored in the user's Verida Wallet. More information is available in the [Polygon ID Verification documentation](https://0xpolygonid.github.io/tutorials/verifier/verifier-overview/).

> There is also a third party product, [Gatekeeper software](https://gatekeeper.software/), that provides a simple, web based, configuration tool to create code that can be embedded in your website to generate a verification request QR code.

Author: ```suggestion
There are also third party products, such as [Gatekeeper software](https://gatekeeper.software/), that provide simple, web based, configuration tools to create code that can be embedded in your website to generate verification requests.
```
